### PR TITLE
fix several warnings

### PR DESCRIPTION
**Pull request by flavianmissi**
mostly invalid escape sequences

### CODE REVIEW

File: buildtrigger/customhandler.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ class CustomBuildTrigger(BuildTriggerHandler):
             "ref": {
                 "type": "string",
                 "description": "git reference for a git commit",
-                "pattern": "^refs\/(heads|tags|remotes)\/(.+)$",
+                "pattern": "^refs/(heads|tags|remotes)/(.+)$",
```
Comment (dmage) [inline buildtrigger/customhandler.py:87]: note that python interprets `"foo\/bar"` as `"foo\\/bar"`, so maybe it needs to be `r"^refs\/(heads|tags|remotes)\/(.+)$"`

File: image/docker/schema2/config.py [hunk 2]
```diff
@@ -64,11 +64,11 @@
   "history": [
     {
       "created": "2018-04-03T18:37:09.284840891Z",
-      "created_by": "\/bin\/sh -c #(nop) ADD file:9e4ca21cbd24dc05b454b6be21c7c639216ae66559b21ba24af0d665c62620dc in \/ "
+      "created_by": "/bin/sh -c #(nop) ADD file:9e4ca21cbd24dc05b454b6be21c7c639216ae66559b21ba24af0d665c62620dc in / "
     },
     {
       "created": "2018-04-03T18:37:09.613317719Z",
-      "created_by": "\/bin\/sh -c #(nop)  CMD [\"sh\"]",
+      "created_by": "/bin/sh -c #(nop)  CMD ["sh"]",
```
Comment (dmage) [inline image/docker/schema2/config.py:71]: `\"` seems to be legit, perhaps it should be `\\"` or the whole string should be `r"..."`

File: buildtrigger/customhandler.py (hunk 1 above)
Comment (flavianmissi) [reply buildtrigger/customhandler.py:87]: I've tested both `r"^refs\/(heads|tags|remotes)\/(.+)$"` and `r"^refs/(heads|tags|remotes)/(.+)$"` and they seem to work the same way.

```python
import jsonschema
from pprint import pprint

schema = {
    "type": "object",
    "properties": {
        "ref": {
            "type": "string",
            "description": "git reference for a git commit",
            "pattern": r"^refs\/(heads|tags|remotes)\/(.+)$",
        },
    },
}

def check(obj, validator):
    pprint(obj)
    result = "VALID" if validator.is_valid(obj) else "INVALID"
    print(f"=> {result}")

validator = jsonschema.Draft4Validator(schema)
check({"ref": "refs/heads/master"}, validator)
# valid
check({"ref": "refs/tags"}, validator)
# invalid

# now without the escape
schema = {
    "type": "object",
    "properties": {
        "ref": {
            "type": "string",
            "description": "git reference for a git commit",
            "pattern": r"^refs/(heads|tags|remotes)/(.+)$",
        },
    },
}

validator = jsonschema.Draft4Validator(schema)
check({"ref": "refs/heads/master"}, validator)
# valid
check({"ref": "refs/tags"}, validator)
# invalid
```

File: image/docker/schema2/config.py (hunk 2 above)
Comment (flavianmissi) [reply image/docker/schema2/config.py:71]: good catch!

Comment (flavianmissi) [reply image/docker/schema2/config.py]: fixed.

File: buildtrigger/customhandler.py (hunk 1 above)
Comment (dmage) [reply buildtrigger/customhandler.py:87]: nice!

**Review by dmage — APPROVED** [review]

**Pull request merged**